symfony/google-mailer/4.4/manifest.json

```diff
@@ -0,0 +1,6 @@
+{
+    "env": {
+        "#1": "Gmail SHOULD NOT be used on production, use it in development only.",
```

Comment:
Should comment to be prefixed with `#`? Also why Gmail is not recommended for prod?

Reply:
it is prefixed (in the name of the env var, which is what goes at the beginning of the line)

Reply:
Gmail is not meant to be used to send emails from an application. Accounts are personal. So, it works for testing on a dev machine, but not for prod.